tests/ci/profile_handler.go

```diff
@@ -109,13 +111,16 @@ func GenerateClusterCreationArgsByProfile(profile *Profile) (clusterArgs *EXE.Cl
 		acctPrefix := clusterArgs.ClusterName
 		majorVersion := ""
 		if profile.Version != "" {
-			majorVersion = strings.Join(strings.Split(profile.Version, ".")[0:2], ".")
+			if profile.Version != "latest" {
+				majorVersion = strings.Join(strings.Split(profile.Version, ".")[0:2], ".")
+			} else {
+				majorVersion = profile.Version
```

Comment:
if version is specified as "latest" - major version should be retrieved by rhcs_versions  or another function, no?

Reply:
fixed